jenkins_pipelines/environments/uyuni-master-qe-build-validation-PRV

```diff
@@ -22,7 +22,7 @@ node('sumaform-cucumber-provo') {
         parameters([
             string(name: 'cucumber_gitrepo', defaultValue: 'https://github.com/uyuni-project/uyuni.git', description: 'Testsuite Git Repository'),
             string(name: 'cucumber_ref', defaultValue: 'master', description: 'Branch prepared for the MU tested'),
-            string(name: 'tf_file', defaultValue: 'susemanager-ci/terracumber_config/tf_files/Uyuni-Master-build-validation.tf', description: 'Path to the tf file to be used'),
+            string(name: 'tf_file', defaultValue: 'susemanager-ci/terracumber_config/tf_files/Uyuni-Master-build-validation-PRV.tf', description: 'Path to the tf file to be used'),
```

Comment:
The title states BV in NUE but the new file name is appended with PRV. This seems wrong to me.

Reply:
I renamed the file to make room for the -NUE one.

Reply:
I see. Thank you for the clarification.